backend/src/routes/getHistoricalBalance.ts

```diff
@@ -0,0 +1,46 @@
+import { sql } from 'drizzle-orm';
+import type { FastifyInstance } from 'fastify';
+import { usdcBalance } from '../db/schema';
+
+const addressRegex = /^0x0[0-9a-fA-F]{63}$/;
```

Comment:
move this in `routes/index.ts`

Reply:
What am I suppose to move here. I don't quite get it

Reply:
move the regex in `routes/index.ts` (as every endpoint uses it) and import it